blue_interface/blue_interface.py

```diff
@@ -236,12 +262,19 @@ def get_gripper_effort(self):
         """
         return self._gripper_effort
 
-    def set_joint_positions(self, joint_positions, duration=0.0):
+    def set_joint_positions(self, joint_positions,
+                            duration=0.0, soft_position_control=False):
```

Comment:
This currently does not do anything?

Reply:
Spooky, maybe I forgot to push something -- will check tonight